process_labels/loading_funcs/tanzania.py

```diff
@@ -100,6 +102,37 @@ def load_tanzania():
     return df
 
 
+def load_tanzania_ceo():
+
+    ceo_files = (DATASET_PATH / "tanzania" / "ceo_labels").glob("*.csv")
+
+    gdfs: List[geopandas.GeoDataFrame] = []
+    for filepath in ceo_files:
+        single_df = pd.read_csv(filepath)
+        single_df[RequiredColumns.GEOMETRY] = single_df["sample_geom"].apply(wkt.loads)
+        single_df["is_crop_mean"] = single_df.apply(
+            lambda x: x["Crop/non-Crop"] == "Cropland", axis=1
+        )
+        gdfs.append(geopandas.GeoDataFrame(single_df, crs="epsg:4326"))
+
+    df = pd.concat(gdfs)
+    df = df.groupby("plotid").agg(
+        {
+            RequiredColumns.LON: "first",
+            RequiredColumns.LAT: "first",
+            RequiredColumns.GEOMETRY: "first",
+            "is_crop_mean": "mean",
+        }
+    )
+
+    df[RequiredColumns.COLLECTION_DATE] = datetime(2019, 1, 2)
+    df[RequiredColumns.EXPORT_END_DATE] = datetime(2022, EXPORT_END_MONTH, EXPORT_END_DAY)
```

Comment:
@adebowaledaniel is this also supposed to be 2022?

Reply:
Yes, the collection year is meant to be 2019! It's an updated Tanzania CEO 2019 dataset. Check the pull request description.

Reply:
yes but the export end date is 2022; should it be 2019?

Reply:
Okay, I get your point. But I thought it was meant to reference the year and date it was exported, which in this case is 2022 (updated), although first 2019. Let me know if you want me to change it to 2019 anyways.